### DISCUSSION TITLE
Pocketbase Adapter

### DISCUSSION THREAD

**Discussion by Teravla** [nextauthjs/next-auth#11118] [Ideas]
### Goals

Add a custom adapter for Pocketbase

### Non-Goals

1.
2.
3.

### Background

I've made an adapter for the db pocketbase but before PR, it says that prior discussion is recommended. 

I tried my best to respect the conditions to include my adapter in the source code. As this is my first participation in an open-source project, I'd be delighted to discuss the ins and outs of this addition. 

For technical information, I've taken my inspiration from the prisma adapter. However, after a few problems with pnpm and ts, I noticed that @auth/core/adapter is not correctly resolved in the index.ts files. The tests I requested were therefore not successful. 

Translated with DeepL.com (free version)

### Proposal

I greatly appreciate the work done by the PocketBase team, which is why I wanted to create a connection between auth and PB. I'd be delighted to help you achieve this link.

**Comment by Niilyx** [comment]
Hopefully this gets added

**Comment by ilbrigz** [comment]
Please add this.

**Comment by 0xsarwagya** [comment]
Our Team Has Created An Adapter Recently Since We Are Using Pocketbase Now.

It's Still Under Active Development and Contributions are Welcome

[You Can look at The Github Repo Here](https://github.com/RebackkHQ/pocketbase-adapter)
[Also The NPM Package Here](https://www.npmjs.com/package/@rebackk/pocketbase-adapter)

**Reply by eadwinCode** [reply]
I see the source code is private. Is this solution not open sourced?

**Reply by 0xsarwagya** [reply]
Open Sourced It :)

The Repo Was Moved Sorry :)

**Reply by ardinusawan** [reply]
> Open Sourced It :)
> 
> The Repo Was Moved Sorry :)

Hi can you share the repo then? Thx in advance
